src/react/src/components/Contribute/SearchByNameAndAddressNotFoundResult.jsx

```diff
@@ -45,7 +45,9 @@ const SearchByNameAndAddressNotFoundResult = ({ classes }) => {
                         root: `${classes.buttonBaseStyles} ${classes.addLocationButtonStyles}`,
                         label: classes.buttonLabelStyles,
                     }}
-                    onClick={() => history.push(productionLocationInfoRoute)}
+                    onClick={() =>
+                        history.push(productionLocationInfoRouteCommon)
+                    }
```

Comment:
Please create a function that handles routing.
onClick={handle...}

Reply:
I've applied separate functions.